src/website/app/demos/Tabs/examples/Tab/basic.js

```diff
@@ -11,14 +11,14 @@ export default {
   scope: { content, Tabs, Tab, Text },
   source: `
     <Tabs label="Lorem ipsum">
-      <Tab title="Tab 1">
-        <Text>{content(1)}</Text>
+      <Tab title="Malachite">
```

Comment:
I like the updates so that the tabs show different content.  It no longer feels broken to me.  Without a fixed height however, as expected, it does make the page jumpy depending on screen size.  Not sure if you want to try to account for this or not...